newroutes/users.ts

```diff
@@ -54,13 +83,14 @@ class UsersRouter {
     return res.sendStatus(200)
   }
 
-  async verifyMobileNumberOtp(req, res) {
+  async verifyMobileNumberOtp(req: Request, res: Response) {
```

Comment:
Why do we not need to use the same RequestHandler we do with verifyEmailOtp?

Reply:
good catch! was using this initially until i realised `express` had a `RequestHandler` type; changed it!